app/views/common_parameters/_form.html.erb

```diff
@@ -8,26 +8,18 @@
         <div class="editor-container">
           <%= render :partial => 'editor/toolbar', :locals => {:show_preview => false} %>
 
-          <% if @common_parameter.hidden_value? %>
-              <%= password_f f, :value, :size => "", :no_label => true, :wrapper_class => "", :value => @common_parameter.value %>
-          <% else %>
-              <%= f.text_area(:value,
-                              :class => "form-control value editor_source",
-                              :rows => line_count(f, :value),
-                              :placeholder => _("Value")) %>
-          <% end %>
+          <%= f.text_area(:value,
+                          :class => "form-control value editor_source #{'masked-input' if @common_parameter.hidden_value?}",
```

Comment:
I find this this is quite complicated expression
`"form-control value editor_source #{'masked-input' if @common_parameter.hidden_value?}"`

maybe something like followning would be more clear
```
<% masked = @common_parameter.hidden_value? ? 'masked-input' : '' %>
<% ... "form-control value editor_source #{masked}" ... %>
```

I suppose it would even work it's kept as statement modifier but I don't like this approach as it's not intuitive why local variable is defined...
```<% masked = 'masked-input' if @common_parameter.hidden_value? %>```